### PR TITLE
Update sync committee stats on dashboard

### PR DESCRIPTION
**Pull request by D13ce**

### CODE REVIEW

File: src/app/pipes/valuestyle.pipe.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export class ValuestylePipe implements PipeTransform {
 		if (args.length == 2) {
 			const secondDrop = args[1] instanceof BigNumber ? args[1] : new BigNumber(args[1])
 			if (value.isGreaterThanOrEqualTo(firstDrop)) return POSITIVE
-			else if (value.isLessThanOrEqualTo(secondDrop)) return WARNING
+			else if (value.isGreaterThanOrEqualTo(secondDrop)) return WARNING
```
Comment (D13ce) [inline src/app/pipes/valuestyle.pipe.ts:39]: Here is how I think these parameters should work:
```
100 - 90% is good
89 - 60% is average
59 - 0% is bad
=>
First Drop: 90
Second Drop: 60
```

Please note that this pipe using two parameters is already used here:
https://github.com/gobitfly/eth2-beaconchain-explorer-app/blob/b55781bcc668a39f525d10ae14ba6a3ea5d7d2a1/src/app/components/dashboard/dashboard.component.html#L531

**Review by stlpruitt314 — APPROVED** [review]

**Review by Hektoreth — APPROVED** [review]

File: src/app/pipes/valuestyle.pipe.ts (hunk 1 above)
Comment (stlpruitt314) [reply src/app/pipes/valuestyle.pipe.ts:39]: Ok

**Review by stlpruitt314 — APPROVED** [review]

**Review by stlpruitt314 — APPROVED** [review]

**Review by stlpruitt314 — APPROVED** [review]

File: src/app/pipes/valuestyle.pipe.ts (hunk 1 above)
Comment (manuelsc) [reply src/app/pipes/valuestyle.pipe.ts:39]: lgtm, seems more intuitive and since we have value that uses the pipe with 2 arguments we can change it.

**Review by manuelsc — APPROVED** [review]
lgtm

**Pull request merged**